deploy/skeletons/__init__.py

```diff
@@ -2,16 +2,12 @@
 # If any other importing logic is wanted, please switch to manual import calls in this file, and remove
 # the dynamic code provided below.
 
-import logging
 import os
 
 for skelModule in os.listdir(os.path.dirname(__file__)):
     if skelModule == "__init__.py" or not skelModule.endswith(".py"):
         continue
 
-    try:
-        __import__(skelModule[:-3], globals(), locals(), level=1)
-    except ImportError:
-        logging.error("Unable to import skeleton %s" % skelModule[:-3])
+    __import__(skelModule[:-3], globals(), locals(), level=1)
```

Comment:
Hmm. I would like to keep this. This is for development sometimes helpful. But of course this shouldn't be a behavior for production.

What do u think about

````py
import viur.core
...

	try:
		__import__(skelModule[:-3], globals(), locals(), level=1)
	except ImportError:
		logging.exception("Unable to import skeleton %s" % skelModule[:-3])
		if not viur.core.config.conf["viur.instance.is_dev_server"]:
			# We accept no import error in the cloud
			raise

Reply:
Hello @sveneberth!

No, I'm not with you. The importing behavior should be the same in local development environment. Multiple times I've got the issue, that import errors where silently dropped, e.g. because of invalid bone names or syntax errors, and it was not recognized both during local development or in the cloud. A hard exception raise in this case will be more helpful.

Reply:
I don't agree with you 100%, but for newbies, a hard raised error may probably make more sense. "Experts" can adapt it as they like, since the base is just a template.